robot-command/src/main/java/org/obolibrary/robot/TemplateCommand.java

```diff
@@ -34,15 +34,19 @@
   /** Initialize the command. */
   public TemplateCommand() {
     Options o = CommandLineHelper.getCommonOptions();
-    o.addOption("i", "input", true, "load ontology from a file");
-    o.addOption("I", "input-iri", true, "load ontology from an IRI");
+    o.addOption("i", "input", true, "load ontology to merge from a file");
+    o.addOption("I", "input-iri", true, "load ontology to merge from an IRI");
     o.addOption("o", "output", true, "save ontology to a file");
     o.addOption("O", "ontology-iri", true, "set the output ontology IRI");
     o.addOption("V", "version-iri", true, "set the output version IRI");
     o.addOption("t", "template", true, "read template from a file");
-    o.addOption("a", "ancestors", false, "MIREOT ancestors into results");
+    o.addOption("A", "ancestors", false, "MIREOT ancestors into results");
```

Comment:
We can't change options that we've published, it could break something downstream.